backend/src/main/java/ca/bc/gov/hlth/hnweb/security/SecurityUtil.java

```diff
@@ -92,4 +95,7 @@ private static String extractOrganization(Jwt jwt) {
         return permissions;
 	}
 
-}
+    @Value("${spring.security.oauth2.resourceserver.jwt.audience}")
+    public void setKeycloakClientStatic(String keycloakClient){
+        SecurityUtil.KEYCLOAK_CLIENT = keycloakClient;
+    }}
```

Comment:
Do we want to put this code right under the variable declaration? Might be clearer what's happening that way.

Reply:
Agreed, moved method up for clarity